### PR TITLE
[1-readme-fixes] Readme updated

### LINKED ISSUE #1: Readme fixes
Worth to update a bit the readme.md file

> Target of project are students with hearing problem and completely deaf.
I will change into:

The project target is helping hearing impaired students during the lessons they attend. But potentially can be used by any student.

> Main objective of the created system is to help those students by showing them transcrypted to text lessons.

The project aims to introduce a speech to text system able to transcript the voice of the professor into a text on the screen of the student pc or mobile phone.

Registered user will have premium feature to store the transcriptions online and save them to a file (txt or pdf format).

# App architecture
It's a bit messy like that, it's more arrows than info.

# Database Architecture
Can you improve that? is a bit too easy and not clear

# App flow
Not clear, avoid user camel case for the page blocks.
I think you should make it more clear and talkative

# Endpoint
I would re-write that in a better way like

User related
| Endpoint | Method | Authenticated | Action |
| - | - | - | - |
| /user | GET | * | Returns current user data |
| /user | POST | - | Creates a new user |
| /user | PUT | * | Updates current user |
| /user/authenticate | GET | - | Authenticate the user, returning access token |

### PR DESCRIPTION
**Pull request by wojciechszmelczerczyk**
## App architecture
-  More info added

## App flow
- I made app flow more talkative

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ Client side will be created in `React` using `Javascript` language.
 git clone repo
 ```
 
+### Navigate to project folder
+
+```sh
+cd /path/to/project
```
Comment (matteocrippa) [inline readme.md]: trivial, at least add the name of the folder for the proejct, i smell it should be express-notes-app

**Review by matteocrippa — CHANGES_REQUESTED** [review]
Drop specific notes in place where some changes are needed

File: readme.md [hunk 2]
```diff
@@ -111,17 +116,17 @@ JWT_EXPIRATION=
...
 
-[![](https://mermaid.ink/img/pako:eNpNj8EKwjAMhl-l5LTB9gI7CFvrwYMgKnjpJVujFm07uu4gY-9uXR0sp3z__yckE3ROEVTw8Ng_2VVIy2LVGX9rsoHdqM1ZWe5Yk51pCKw-HfJ_ZJF5gmaBOgHfQnJ4dtSdd4O7b1ckT2QCA7Y4UL6dbxKIFaSFAgx5g1rFe6efLSE8yZCEKrYK_UuCtHPMjb3CQHulg_NQBT9SATgGd_nYbuWUERrj6yaJ8xdH-E18)](https://mermaid.live/edit#pako:eNpNj8EKwjAMhl-l5LTB9gI7CFvrwYMgKnjpJVujFm07uu4gY-9uXR0sp3z__yckE3ROEVTw8Ng_2VVIy2LVGX9rsoHdqM1ZWe5Yk51pCKw-HfJ_ZJF5gmaBOgHfQnJ4dtSdd4O7b1ckT2QCA7Y4UL6dbxKIFaSFAgx5g1rFe6efLSE8yZCEKrYK_UuCtHPMjb3CQHulg_NQBT9SATgGd_nYbuWUERrj6yaJ8xdH-E18)
+[![](https://mermaid.ink/img/pako:eNptkLFqw0AMhl9FaHIgfgEPBdvXoUOhtIUutyg-JT4S37l3uiHEefeqJB5SqkXi14d-SRccomNs8JBoHuHT2AAabdWfPAeBL95toK6fYEn8XTjLAl31rhnat5fNDe5WIM8xZF6gfdAPLJBn5mGsj3xeoK9e_ZBijvt_hqhHUsZUhoR2lPne7lcPKSk8jOtugPmzhMo24BYnThN5pxdefkGLMvLEFhstHaWjRRuuypXZkfCz8xITNns6Zd4iFYkf5zBgI6nwChlP-q3pTl1_AC3UZ08)](https://mermaid.live/edit#pako:eNptkLFqw0AMhl9FaHIgfgEPBdvXoUOhtIUutyg-JT4S37l3uiHEefeqJB5SqkXi14d-SRccomNs8JBoHuHT2AAabdWfPAeBL95toK6fYEn8XTjLAl31rhnat5fNDe5WIM8xZF6gfdAPLJBn5mGsj3xeoK9e_ZBijvt_hqhHUsZUhoR2lPne7lcPKSk8jOtugPmzhMo24BYnThN5pxdefkGLMvLEFhstHaWjRRuuypXZkfCz8xITNns6Zd4iFYkf5zBgI6nwChlP-q3pTl1_AC3UZ08)
```
Comment (matteocrippa) [inline readme.md:119]: Make more clear the registration flow, there's something i don't get in the retry

File: readme.md [hunk 3]
```diff
@@ -58,7 +58,13 @@ Client side will be created in `React` using `Javascript` language.
 ### Clone repository
 
 ```
-git clone repo
+git clone https://github.com/wojciechszmelczerczyk/express-notes-app.git
```
Comment (wojciechszmelczerczyk) [inline readme.md:61]: project url added to project clone readme section

File: readme.md [hunk 4]
```diff
@@ -58,7 +58,13 @@ Client side will be created in `React` using `Javascript` language.
...
+```
+
+### Navigate to project folder
+
+```sh
+cd /express-notes-app
```
Comment (wojciechszmelczerczyk) [inline readme.md:67]: replaced arbitrary path with project name

File: readme.md [hunk 5]
```diff
@@ -111,17 +116,17 @@ JWT_EXPIRATION=
...
+[![](https://mermaid.ink/img/pako:eNpVks9uwjAMxl_FyolJ8AI9bALa8ee63SgHq_FKBE0q1zChlnefSwp0vaT5_LNjf0lrimDJJKZkrA_wneYe9JvvmErXCDHUWNIeZrP3ruZwcZagYLLkxeGp6WDRIv-ToAjMVMjHLZZa9LmwmVypeRsr24kPg7C9C53wFZ7HYonOdzCPxCYSHi-uRCGQAJXG7811sNw9N_vILyOvbfW0p1_wQZRMd_0K4uREYz6NfEMCL6CDLOIjMBtAQRZtVme1zpdD9c_WNTG_wQvZhwOf95zVyIGorF8OrGJZddH13vXzndQHHS3G1zF-JKpHpwY1KMu9mZqKWC2weo9tn5AbOVBFuUn01yIfc5P7m3Ln2qohmXUS2CQ_el00NXiW8HX1hUmEz_SAUof6JqqBuv0B3yOyKw)](https://mermaid.live/edit#pako:eNpVks9uwjAMxl_FyolJ8AI9bALa8ee63SgHq_FKBE0q1zChlnefSwp0vaT5_LNjf0lrimDJJKZkrA_wneYe9JvvmErXCDHUWNIeZrP3ruZwcZagYLLkxeGp6WDRIv-ToAjMVMjHLZZa9LmwmVypeRsr24kPg7C9C53wFZ7HYonOdzCPxCYSHi-uRCGQAJXG7811sNw9N_vILyOvbfW0p1_wQZRMd_0K4uREYz6NfEMCL6CDLOIjMBtAQRZtVme1zpdD9c_WNTG_wQvZhwOf95zVyIGorF8OrGJZddH13vXzndQHHS3G1zF-JKpHpwY1KMu9mZqKWC2weo9tn5AbOVBFuUn01yIfc5P7m3Ln2qohmXUS2CQ_el00NXiW8HX1hUmEz_SAUof6JqqBuv0B3yOyKw)
```
Comment (wojciechszmelczerczyk) [inline readme.md:127]: registration flow updated

**Review by matteocrippa — APPROVED** [review]

**Pull request merged**